src/test/java/StringCalculatorTest.java

```diff
@@ -0,0 +1,47 @@
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+class StringCalculatorTest {
+
+    @ParameterizedTest
+    @CsvSource(value = {"2 + 3 * 4 / 2,10", "2 * 6,12", "5 - 3 * 2,4"})
+    void 계산을_할_수_있다(String expression, int expect) {
+        //given
+        StringCalculator calculator = new StringCalculator();
+
+        //when
+        int result = calculator.calculate(expression);
+
+        //then
+        assertThat(result).isEqualTo(expect);
+    }
+
+    @Test
+    void 입력_값이_null일_수_없다() {
+        //given
+        StringCalculator calculator = new StringCalculator();
+
+        //when
+
+        //then
+        assertThatExceptionOfType(IllegalArgumentException.class)
+                .isThrownBy(() -> calculator.calculate(null)).withMessageContaining("입력 값이 null 혹은 빈 문자열입니다.");
```

Comment:
이렇게 메서드 체이닝으로 연결되는 경우 `.`는 한 줄에 하나씩만 위치하도록 구성하는 것이 좋음.
```suggestion
        assertThatExceptionOfType(IllegalArgumentException.class)
                .isThrownBy(() -> calculator.calculate(null))
                .withMessageContaining("입력 값이 null 혹은 빈 문자열입니다.");
```